packages/seedless-onboarding-controller/src/SeedlessOnboardingController.ts

```diff
@@ -1856,6 +1875,81 @@ export class SeedlessOnboardingController<EncryptionKey> extends BaseController<
     });
   }
 
+  /**
+   * Revoke all pending refresh tokens.
+   *
+   * This method is to be called after user is authenticated.
+   */
+  async revokePendingRefreshTokens() {
+    this.#assertIsAuthenticatedUser(this.state);
+    const { pendingToBeRevokedTokens } = this.state;
+    if (!pendingToBeRevokedTokens || pendingToBeRevokedTokens.length === 0) {
+      return;
+    }
+
+    // revoke all pending refresh tokens in parallel
+    const promises = pendingToBeRevokedTokens.map(({ revokeToken }) => {
```

Comment:
#removePendingRefreshToken remove all refresh token related to revoke token locally but we are looping all tokens pairs here?

Should we group to same revoke token and just call 1 api call for the same revoke token